kernel/nativecode.ml

```diff
@@ -1841,7 +1841,7 @@ and apply_fv env sigma univ (fv_named,fv_rel) auxdefs ml =
   in
   let auxdefs = List.fold_right get_rel_val fv_rel auxdefs in
   let auxdefs = List.fold_right get_named_val fv_named auxdefs in
-  let lvl = Context.Rel.length env.env_rel_context.env_rel_ctx in
+  let lvl = Context.Rel.length (rel_context env) in
```

Comment:
I think I prefer the original code in this file. In fact, I believe we could remove the aliases of the projections, but it's ok if we don't do it now.

Reply:
>I think I prefer the original code in this file. In fact, I believe we could remove the aliases of the projections, but it's ok if we don't do it now.

Why?

Reply:
This argument was made before, but we tend to have a lot of useless aliases that make the code base larger and harder to browse. If they have no point, why keep them?

Reply:
Here going through a function is simpler than going through 2 projection layers.

Reply:
Ok, keep this one if you prefer it.